exporters/jaeger/src/test/java/io/opentelemetry/exporters/jaeger/AdapterTest.java

```diff
@@ -86,7 +86,7 @@ void testProtoSpan() {
     assertEquals(Timestamps.fromMillis(startMs), jaegerSpan.getStartTime());
     assertEquals(duration, Durations.toMillis(jaegerSpan.getDuration()));
 
-    assertEquals(4, jaegerSpan.getTagsCount());
+    assertEquals(5, jaegerSpan.getTagsCount());
```

Comment:
It might be nice to have a test that verifies that we have 6 keys when the version is set, as well.

Reply:
but I won't hold up this PR for it, since we have higher-level tests with the details in them already